### PR TITLE
Use compound path notation in help-text

### PR DESCRIPTION
**Pull request by SimonBarendse**

### CODE REVIEW

File: internals/secrethub/path_placeholders.go [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package secrethub
+
+const (
+	repoPathPlaceHolder                  = "<namespace>/<repo>"
+	dirPathPlaceHolder                   = repoPathPlaceHolder + "/<dir>[/<dir> ...]"
+	optionalDirPathPlaceHolder           = repoPathPlaceHolder + "[/<dir> ...]"
+	secretPathPlaceHolder                = optionalDirPathPlaceHolder + "/<secret>"
+	secretPathOptionalVersionPlaceHolder = secretPathPlaceHolder + "[:<version>]"
+	anyPathPlaceHolder                   = "<namespace>[/<repo>[/<dir> ...][/<secret>[:<version>]]]"
```
Comment (jpcoenen) [inline internals/secrethub/path_placeholders.go]: Though this is technically correct, I don't think it makes things a lot more understandable. Can we make this simpler?

File: internals/secrethub/repo_init.go [hunk 2]
```diff
@@ -27,7 +27,7 @@ func NewRepoInitCommand(io ui.IO, newClient newClientFunc) *RepoInitCommand {
 // Register registers the command, arguments and flags on the provided Registerer.
 func (cmd *RepoInitCommand) Register(r command.Registerer) {
 	clause := r.Command("init", "Initialize a new repository.")
-	clause.Arg("repo-path", "Path to the new repository (<namespace>/<repo>)").Required().SetValue(&cmd.path)
+	clause.Arg("repo-path", "Path to the new repository.").Required().PlaceHolder(repoPathPlaceHolder).SetValue(&cmd.path)
```
Comment (jpcoenen) [inline internals/secrethub/repo_init.go]: Not a full sentence, so there should not be an ending `.`.

File: internals/secrethub/path_placeholders.go (hunk 1 above)
Comment (SimonBarendse) [reply internals/secrethub/path_placeholders.go]: How about just using `secrethub ls [<path>]`?

Comment (jpcoenen) [reply internals/secrethub/path_placeholders.go]: Sounds a lot better :+1:

**Pull request merged**